packages/ts-sdk/src/wallet/onchain.ts

```
@@ -302,6 +259,76 @@ export class OnchainWallet implements AnchorBumper {
             return [parent.hex, child.hex];
```

Biome (2.5.1)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)